x11-misc/xkeyboard-config/xkeyboard-config-2.39.ebuild

```diff
@@ -19,21 +21,33 @@ fi
 
 LICENSE="MIT"
 SLOT="0"
+IUSE="man"
 
 DEPEND=""
 RDEPEND=""
 BDEPEND="
 	${PYTHON_DEPS}
 	dev-lang/perl
-	dev-libs/libxslt
+	man? ( dev-libs/libxslt )
 	sys-devel/gettext
 "
 
+src_prepare() {
+	default
+	plocale_get_locales > po/LINGUAS || die
+}
+
 pkg_setup() {
 	python-any-r1_pkg_setup
 }
 
 src_configure() {
+	if ! use man; then
+		# Make sure xslt is disabled
+		sed -i -e 's/if xsltproc\.found()/if false/' 'meson.build'
+	fi
+	# Make sure pytest is disabled
```

Comment:
The `tests/` directory in the upstream repo looks like it uses pytest. Let's wire up the tests instead?

Reply:
That's where it's either better to file a bug or put a big TODO - we don't want anyone to accidentally merge it thinking it's right. We found an issue (automagic pytest). Just disabling it isn't a fix unless we think it through.